weldx/schemas/weldx.bam.de/weldx/core/geometry/spatial_data-1.0.0.yaml

```diff
@@ -38,7 +38,7 @@ properties:
       datatype:
         type: string
         enum: [float32, float64]
-    wx_shape: [(:), 3]
+    wx_shape: [(~), 3]
```

Comment:
I have changes this because `:` is not a valid yaml syntax in this context (it got picked up by the linter)

I have already removed the mention of `:` in [the documentation](https://weldx.readthedocs.io/en/latest/standard/shape-validation.html) but I think we might also just remove the support in the code as well? @vhirtham @mark141